release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -688,6 +688,13 @@ The Operator SDK's `run bundle-upgrade` subcommand automates triggering an insta
 
 For more information, see xref:../operators/operator_sdk/osdk-working-bundle-images.adoc#osdk-bundle-upgrade-olm_osdk-working-bundle-images[Testing an Operator upgrade on Operator Lifecycle Manager].
 
+[id="ocp-4-8-osdk-compat"]
+==== Controlling Operator compatibility with {product-title} versions
+
+When an API is removed from an {product-title} version, Operators running on that cluster version that are still using removed APIs will no longer work properly. As an Operator author, you should plan to update your Operator projects to accommodate API deprecation and removal to avoid interruptions for users of your Operator.
```

Comment:
s/removed APIs/the removed APIs

Reply:
Deprecated != removed, so we're trying to keep the distinctions clear.